pages/administration/productions/[productionSlug]/bookings/[bookingReference]/index.vue

```diff
@@ -16,34 +16,45 @@
       </table>
     </UiCard>
     <UiCard title="Ticket Inspection" class="mb-4">
-      <table>
-        <table-row>
-          <table-head-item>Ticket ID</table-head-item>
-          <table-head-item>Seat Group</table-head-item>
-          <table-head-item>Concession Type</table-head-item>
-          <table-head-item>Checked In</table-head-item>
-          <table-head-item v-if="anyTicketsChecked"
-            >Checked In By</table-head-item
+      <UiTablesPaginatedTable
+        v-model:offset="ticketInspectionOffset"
+        :items="booking.tickets"
+        :max-per-page="10"
+        :page-info="{}"
```

Comment:
If you aren't using these features I'd just remove them

Reply:
`offset`, `items` and `page-info` all seem required by the component, and the page crashes out without them: I can only remove `max-per-page` without issue...

Reply:
Ok fair enough - this component is meant to help you implement pagination very easily. Probably won't run into that issue here so I don't really mind. Just hard-code the offset as 0 (better not to register a reactive variable if we don't have to for performance!): `v-model:offset` -> `:offset="0"`